Src/Pretty.hs

```diff
@@ -65,28 +76,37 @@ instance Pretty () where
 instance Pretty Void where
   pretty = absurd
 
+------------------------------------------------------------------
+-- | a 't's worth of |Doc| can be 'Collapse'd if it can be flattened to a 'Doc' 
 class Collapse t where
   collapse :: t (Doc Annotations) -> Doc Annotations
 
-newtype BracesList t = BracesList { unBracesList :: [t] }
-
-instance Collapse BracesList where
-  collapse (BracesList []) = "{}"
-  collapse (BracesList ds) = encloseSep "{" "}" "; " ds
-
+-- | print snoc lists as "< a , b , c ]", and the empty one as "[<]"
 instance Collapse Bwd where
   collapse B0 = "[<]"
   collapse ds =  encloseSep "<" "]" ", " (ds <>> [])
```

Comment:
Is this a typo? Surely we meant to print `[<3,2,1]`

Reply:
I would think so as well... I'm just documenting the code as it exists!